src/GitHub/Data/Content.hs

```diff
@@ -87,3 +155,74 @@ instance FromJSON ContentInfo where
                 <*> o .: "url"
                 <*> o .: "git_url"
                 <*> o .: "html_url"
+
+instance FromJSON ContentResultInfo where
+  parseJSON = withObject "ContentResultInfo" $ \o ->
+    ContentResultInfo <$> parseJSON (Object o)
+                  <*> o .: "size"
+
+instance FromJSON ContentResult where
+  parseJSON = withObject "ContentResult" $ \o ->
+    ContentResult <$> o .: "content"
+                  <*> o .: "commit"
+
+instance ToJSON Author where
+  toJSON (Author { authorName   = name
+                 , authorEmail  = email
+                 }) = object
+                 [ "name"      .= name
+                 , "email"     .= email
+                 ]
+
+instance ToJSON CreateFile where
+  toJSON (CreateFile { createFilePath       = path
```

Comment:
I'd use `RecordWildCards`, this is extremely ugly. (No need to touch old code though)